client/src/components/Question.js

```diff
@@ -1,28 +1,44 @@
 import React from 'react';
-import { Form, Jumbotron } from 'react-bootstrap';
+import { Form, Card } from 'react-bootstrap';
 
 const Question = ({ question, handleAnswerSelect, selectedAnswer = [] }) => {
+  const addClassName = question => {
+    if (question.qType === 'checkbox') {
+      return 'styled-input-square';
+    } else {
+      return 'styled-input-rounded';
+    }
+  };
+
   return (
     <fieldset>
-      <Jumbotron>
-        <h2>{question.text}</h2>
-        <Form.Group key={question._id}>
-          {question.answers.map(answer => {
-            return (
-              <Form.Check
-                key={answer._id}
-                type={question.qType}
-                id={`${question._id}.${answer._id}`}
-                name={question._id}
-                value={answer._id}
-                onChange={e => handleAnswerSelect(e, question.qType)}
-                label={`${answer.text}`}
-                defaultChecked={selectedAnswer.includes(answer._id)}
-              />
-            );
-          })}
-        </Form.Group>
-      </Jumbotron>
+      <Card className="mb-3 rounded">
+        <Card.Header className="bg-secondary text-white" as="h5">
+          {question.text}
+        </Card.Header>
+        <Card.Body className="bg-white">
+          <Form.Group className="bg-light px-1" key={question._id}>
+            {question.answers.map(answer => {
+              return (
+                <div className="border-bottom border-white">
+                  <Form.Check
+                    key={answer._id}
+                    type={question.qType}
+                    className={addClassName(question)}
+                    id={`${question._id}.${answer._id}`}
+                    name={question._id}
+                    value={answer._id}
+                    onChange={e => handleAnswerSelect(e, question.qType)}
+                    label={`${answer.text}`}
```

Comment:
```suggestion
                    label={answer.text}
```
Since we are not concatenating strings, we don't need to put `answer.text` in a string template